e2e/staking-pool.e2e.ts

```diff
@@ -0,0 +1,185 @@
+import { KeyTags } from "@ew-did-registry/did-resolver-interface";
```

Comment:
Duplication with staking.pool.e2e.ts. can be avoided if combine all tests and include them in two describe blocks, which sets up different staking pool factories

Reply:
IMO there is no point in combining them both as we will remove older staking, old staking code was kept for backward-compatibility purposes. This staking is quite different than previous one.